src/test/ovmsconfig_test.cpp

```diff
@@ -224,13 +224,23 @@ TEST_F(OvmsConfigDeathTest, missingParams) {
 TEST_F(OvmsConfigDeathTest, negativePortMin) {
     char* n_argv[] = {"ovms", "--config_path", "/path1", "--port", "-1"};
     int arg_count = 5;
-    EXPECT_EXIT(ovms::Config::instance().parse(arg_count, n_argv), ::testing::ExitedWithCode(OVMS_EX_USAGE), "error parsing options: Argument ‘-1’");
+#ifdef __linux__
+    std::string error = "‘-1’";
```

Comment:
Those exit codes are not connected to OVMS - if you compare:
grep -R -n EX_USAGE | grep -v test
main_benchmark.cpp:112:        exit(EX_USAGE);
main_benchmark.cpp:357:        return EX_USAGE;
main_benchmark.cpp:400:        return EX_USAGE;
server.cpp:351:        return EX_USAGE;
cli_parser.cpp:186:        exit(EX_USAGE);
cli_parser.cpp:216:        exit(EX_USAGE);
config.cpp:54:        exit(EX_USAGE);
ovms_exit_codes.hpp:30:#define OVMS_EX_USAGE EX_USAGE
ovms_exit_codes.hpp:35:#define OVMS_EX_USAGE 3

To:
grn exit_codes.hpp
test/ovmsconfig_test.cpp:27:#include "ovms_exit_codes.hpp"
BUILD:422:        "ovms_exit_codes.hpp",

Seems like exit codes were not used in actual OVMS.

Reply:
Yes, this is just a fast define for ovms_test. There is another task to write proper exit_codes for ovms with ntstatus.h